fineract-provider/src/main/resources/sql/migrations/core_db/V1020__velocity_update_vnf_account_dates_to_31Dec2021.sql

```diff
@@ -0,0 +1,50 @@
+--
```

Comment:
@hiteshprajapati why are these separate migrations?

Reply:
@nassan : The Fineract uses the flyway(https://flywaydb.org/) for these migration. 
If we want to change any existing migration sql that we already have and already run on db, then we need to add seperate migraion SQL for that rather than changing existing SQL. If we change the existing SQL, the flyway will fail the migration during the fineract startup due to change in checksum of existing script which already executed.